docs/modules/ROOT/pages/getting-spring-security.adoc

```diff
@@ -65,13 +98,27 @@ You can do so by adding a Maven property:
 </properties>
 ----
 
+Gradle::
++
+.build.gradle
+[source,groovy]
+[subs="verbatim,attributes"]
+----
+ext['spring.version']='{spring-core-version}'
+----
+======
+
 If you use additional features (such as LDAP, OAuth 2, and others), you need to also include the appropriate xref:modules.adoc#modules[Project Modules and Dependencies].
 
 [[getting-maven-no-boot]]
-=== Maven Without Spring Boot
+=== Standalone Usage (Without Spring Boot)
 
-When you use Spring Security without Spring Boot, the preferred way is to use Spring Security's BOM to ensure that a consistent version of Spring Security is used throughout the entire project. The following example shows how to do so:
+When you use Spring Security without Spring Boot, the preferred way is to use Spring Security's BOM to ensure that a consistent version of Spring Security is used throughout the entire project.  For Gradle, you can do so by using the https://github.com/spring-gradle-plugins/dependency-management-plugin[Dependency Management Plugin]:
```

Comment:
I'd recommend leaving this link to the plugin until after the snippet and also inside a `TIP`:

```
[TIP]
Spring provides a `Dependency Management Plugin`(link) for Gradle
```

I'd recommend doing the same with the same link later on in the article.